doc/bc/changes-1.0.md

```diff
@@ -262,6 +262,13 @@ Changes affecting version compatibility with deprecated ezpublish-kernel version
 
 * Legacy (SQL) Search Engine will no longer treat deprecated `%` as a search wildcard. Use `*` instead.
 
+* Field Type External Storage Handlers `$context` array no longer has the "connection" key. Rely on
+  injected Connection instead.
+
+* The deprecated Zeta Components (eZc) Database handler has been dropped. All classes and interfaces
+  from the namespaces `eZ\Publish\Core\Persistence\Database` and `eZ\Publish\Core\Persistence\Doctrine`
+  were removed.
+
 ## Deprecated features
 
```

Comment:
There's array context of FieldStorage deprecated. Not a feature per se, but may be worth noting here.

Reply:
Added via f3f4eb2